src/Utils/__tests__/time.test.ts

```diff
@@ -0,0 +1,32 @@
+import { getOffsetBetweenGravityClock } from "../time"
+
+jest.mock("../metaphysics", () => ({ metaphysics: jest.fn() }))
+import { metaphysics } from "../metaphysics"
+const mockphysics = metaphysics as jest.Mock<any>
+
+const SECONDS = 1000
+const MINUTES = 60 * SECONDS
+
+const dateNow = 1525983752000 // Thursday, May 10, 2018 8:22:32.000 PM UTC in milliseconds
+
+it("returns an offset between current time and system time", async () => {
+  jest.useFakeTimers()
+  Date.now = () => dateNow
+
+  // Set up a situation where the client's clock is ahead of Gravity's clock by 10 minutes.
+  mockphysics.mockReturnValueOnce(
+    Promise.resolve({
+      data: {
+        system: {
+          time: {
+            unix: (dateNow - 10 * MINUTES) * 1e-3,
```

Comment:
Why do we need to `* 1e-3`?

Reply:
The units we're dealing with are ms (`dateNow`, `MINUTES`), but this is 'unix time' (aka # of seconds), so we divide by 1000 in this instance.

Reply:
oh for some reason I thought unix epoch timestamps are always ms.